### PR TITLE
Added hindi playlist for Next JS course

### PR DESCRIPTION
**Pull request by sumitmukharjeeeeee**
## What does this PR do?
Add resource

This is the playlist from youtube for a free course on NextJS. Next.js is an open-source development framework built on top of Node.js enabling React-based web applications functionalities such as server-side rendering and generating static websites.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * [Java](#Java)
 * [Javascript](#Javascript)
   * [React](#react)
+* [Next.Js](#NextJS)
```
Comment (SethFalco) [inline courses/free-courses-hi.md:11]:
```suggestion
* [Next.js](#nextjs)
```

It's Next.js, not Next.Js.

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -66,6 +67,10 @@
 * [ReactJS Tutorial in Hindi 2020](https://www.youtube.com/playlist?list=PLwGdqUZWnOp3aROg4wypcRhZqJG3ajZWJ) - Vinod Bahadur Thapa (Thapa Technical)
 
 
+### Next.Js
```
Comment (SethFalco) [inline courses/free-courses-hi.md:70]:
```suggestion
### Next.js
```

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -66,6 +67,10 @@
...
+
+* [NextJS Tutorial in Hindi](https://www.youtube.com/playlist?list=PLwGdqUZWnOp2rDbpfKAeUi9f8qZMS7_cv) - Vinod Bahadur Thapa (Thapa Technical)
+
```
Comment (SethFalco) [inline courses/free-courses-hi.md:73]: There should be an additional line break here.

You can click on the failed build to get additional information on why it's failing. Then you can see exactly what's wrong and how to fix it.

**Review by SethFalco — CHANGES_REQUESTED** [review]

File: courses/free-courses-hi.md [hunk 4]
```diff
@@ -66,6 +67,10 @@
 * [ReactJS Tutorial in Hindi 2020](https://www.youtube.com/playlist?list=PLwGdqUZWnOp3aROg4wypcRhZqJG3ajZWJ) - Vinod Bahadur Thapa (Thapa Technical)
 
 
+### Next.Js
+
+* [NextJS Tutorial in Hindi](https://www.youtube.com/playlist?list=PLwGdqUZWnOp2rDbpfKAeUi9f8qZMS7_cv) - Vinod Bahadur Thapa (Thapa Technical)
```
Comment (SethFalco) [inline courses/free-courses-hi.md:72]:
```suggestion
* [Next.JS Tutorial In Hindi 2021](https://www.youtube.com/playlist?list=PLwGdqUZWnOp2rDbpfKAeUi9f8qZMS7_cv) - Vinod Bahadur Thapa (Thapa Technical)
```

We also prefer to use the name as it appears at the source location, could this be changed to match too!

**Comment by sumitmukharjeeeeee** [comment]
I closed this as Was not able to change the line break commit from this branch.
Please help me how to proceed

**Pull request closed without merging**